app/setup/plugins/modal.js

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
- [ ] I would recommend to rename this to `modalStack` or similar, since it is not really a modal.